### PR TITLE
added tab bar

### PR DESCRIPTION
**Pull request by TimofeyLuk**

### CODE REVIEW

File: PetProject/Coordinators/AppCoordinator.swift [hunk 1]
```diff
@@ -27,24 +30,54 @@ final class AppCoordinator: Coordinator {
...
         let loginVC = factory.loginViewController()
         loginVC.delegate = self
-        navigationController.pushViewController(loginVC, animated: false)
+        loginVC.modalPresentationStyle = .fullScreen
+        tabBarController.setViewControllers([mainNavigationController], animated: false)
+        mainNavigationController.pushViewController(loginVC, animated: true)
     }
     
+    func showAlert(_ alert: UIAlertController) {
+        guard
```
Comment (DzmitryHerasiuk) [inline PetProject/Coordinators/AppCoordinator.swift]: Will it work?
```
let selectedViewController = tabBarController.selectedViewController
selectedViewController?.present(alert, animated: true)
```

File: PetProject/Coordinators/AppCoordinator.swift [hunk 2]
```diff
@@ -27,24 +30,54 @@ final class AppCoordinator: Coordinator {
...
-        navigationController.viewControllers.removeAll()
-        navigationController.pushViewController(mainViewController, animated: true)
+
+        mainNavigationController.viewControllers.removeAll()
+        mainNavigationController.tabBarItem = UITabBarItem(
+            title: "Stores".localized,
+            image: UIImage(systemName: "cart"),
+            selectedImage: UIImage(systemName: "cart.fill")
+        )
+        mainNavigationController.viewControllers.removeAll()
```
Comment (DzmitryHerasiuk) [inline PetProject/Coordinators/AppCoordinator.swift:58]: 55 - 61 lines - the same

File: PetProject/Coordinators/AppCoordinator.swift (hunk 1 above)
Comment (TimofeyLuk) [reply PetProject/Coordinators/AppCoordinator.swift]: Done

File: PetProject/Coordinators/AppCoordinator.swift (hunk 2 above)
Comment (TimofeyLuk) [reply PetProject/Coordinators/AppCoordinator.swift:58]: Done

**Pull request merged**